packages/eslint-config/package.json

```diff
@@ -27,7 +27,7 @@
   "peerDependencies": {
     "@endo/eslint-plugin": "^2.2.3",
     "@jessie.js/eslint-plugin": "^0.4.1",
-    "typescript-eslint": "^7.13.1",
+    "typescript-eslint": "^8.17.0",
```

Comment:
We're getting a lot of duplication because of this major bump. Any way to get rid of the v7 in any way? Looks like the only dependency is `eslint-plugin-github` whose [latest version](https://github.com/github/eslint-plugin-github/blob/v5.1.3/package.json#L38) has bumped to 8 as well.